index.html

```diff
@@ -8,343 +8,328 @@
 	  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     <title>Supreeti Portfolio</title>
   </head>
-  <body>
-    <header class="header1">
+  <body class="bg1">
+    
+    <header class="bg2">
+    <div class="header1">
         <h3>My Logo</h3>
-          <nav>
+          <nav class="nav1">
               <img src="images/Union.png" alt=""/>
           </nav>
-    </header>
+          <nav class="nav2">
+            <ul class="nav2 liststyle">
+                <li class="list_style"> <a class="textdecoration" href="#">Portfolio</a></li>
+                <li class="list_style"> <a class="textdecoration" href="#">About</a></li>
+                <li class="list_style"><a class="textdecoration" href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </div>
+     </header>
+
     <main>
```

Comment:
[OPTIONAL]

- Kindly ensure that your headline section matches the Figma design. Currently the contents are not properly organized 👍🏽 

YOur design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213883077-8442d2ee-64f7-4429-a97b-1b518d8a04cf.png)

FIgma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213883111-2dabf909-230d-4996-9baf-a34a145e9056.png)